src/components/Confirm/index.js

```diff
@@ -21,12 +21,17 @@ const FileViewer = ({file:{preview, type}}) =>
     </embed>
   </object>
 
-
-const CaptureViewerPure = ({capture:{blob}}) =>
+const CaptureViewerPure = ({capture:{blob, base64}}) =>
```

Comment:
What's the point of `CaptureViewerPure`? Why not put all this in the render method of `CaptureViewer`?

Can we just pass `(blob, base64)`?